src/SwedbankPay.Sdk/Payments/CardPayments/CardPaymentsResource.cs

```diff
@@ -23,5 +23,19 @@ public Task<CardPayment> Create(CardPayments.CardPaymentRequest paymentRequest,
         {
             return CardPayment.Create(paymentRequest, this.httpClient, GetExpandQueryString(paymentExpand));
         }
+
+        public Task<CardVerify> Verify(CardPaymentVerifyRequest paymentVerifyRequest, PaymentExpand paymentExpand = PaymentExpand.None)
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5742807)